test/cql-pytest/test_describe.py

```diff
@@ -591,6 +619,7 @@ def test_type_quoting(cql):
         assert f"CREATE TYPE \"{ks_name}\".\"{name}\"" in desc
         assert f"CREATE TYPE \"{ks_name}\".{name}" not in desc
         assert f"CREATE TYPE {ks_name}.\"{name}\"" not in desc
+        # This was buggy in Cassandra - see CASSANDRA-19019
```

Comment:
Looks like this was fixed in a later version.

Reply:
Yes, I see now, thanks. I'll fix the comment / marker. Although it's a bit frustrating the test will fail on older Cassandra, we shouldn't be in the business of tracking Cassandra bugs in our test suite, we can just assume it's run on a recent version of Cassandra.

Reply:
I upgraded my Cassandra to the latest version of Cassandra 4 (4.1.3), and indeed, it works there. I'm changing the test accordingly.

Reply:
Maybe add a note about this to `test/cql-pytest/README.md` (in a follow-up PR). Also, in general, I'm missing a "Running the tests against Cassandra" section in said file anyway.

Reply:
I do have a few sentences about Cassandra spread around the README.md, and even in the "philosophy" section a whole part (5) about "Run your test against Cassandra". But maybe I should split that part into a separate section, higher up the file?

I could also explain how to compile Cassandra on modern Linux distros, which isn't trivial because of the Java 17 mess (although you can also take a pre-compiled version of Java and not deal with the compilation).

Reply:
I looked again at the text of test/cql-pytest/README.md and saw that almost everything I could possibly say about Cassandra is already written there. Except how to actually compile Cassandra. Could you please tell me a bit more what you were missing there? Or maybe the existing text was just hard for you to find among all the other explanations? Or maybe you were missing the part I admit is missing - how to compile Cassandra?

Reply:
When I had a quick look earlier today, I saw a big block of text, and no header standing out, mentioning "Cassandra". Indeed as I read into the text, I do see you talk about it already, so I guess it is fine.

> how to compile Cassandra?

I don't think this is important to describe there, people can find out about that on the C* documentation I suppose.